### DISCUSSION TITLE
pBTC stuck on Solana Wormhole

### DISCUSSION THREAD

**Discussion by owaynewo** [wormhole-foundation/wormhole#229] [Problems / bugs] [answered]
Hi team,

I tried to use wormhole to wrap my Ethereum pBTC to Solana. 
However I can't use the wrapped pBTC on Saber AAM but I can't seem to use wormhole to transfer the pBTC back to Ethereum network. "I think the wormhole frontend UI does't recognised the minted wormhole warpped pBTC (token address: 3oD7PhZstGGxcpLfxsZW8iGby8DQjSB3SJy74NPVHhgT)

https://explorer.solana.com/tx/4yNbZNi7q4v1a7DUFzorbzyxUwerAKppfTSi1uNiY69Qbe9sGa85FiEC8UggTMFK9eMnJ2Uz8dPqL3onFtqsY3zW

^this is the trxn hash

Please kindly advise how I can unwrap the pBTC token back to the Ethereum network.

Many thanks,
Wayne

**Comment by hendrikhofstadt** [comment, marked as answer]
Looks like you managed to transfer them out.
